packages/samples/test/output/nullable/openapi.json

```diff
@@ -36,16 +36,7 @@
           "content": {
             "application/json": {
               "schema": {
-                "type": "object",
-                "properties": {
-                  "num": {
-                    "type": "integer",
-                    "format": "int32"
-                  }
-                },
-                "required": [
-                  "num"
-                ],
+                "$ref": "#/components/schemas/AnotherModel",
```

Comment:
This is technically not valid OpenAPI3, We allow that in autorest but don't think the OpenAPI3 emitter should be doing that

Reply:
Not sure why it's invalid, I saw the a similar example in the https://swagger.io/specification/
![image](https://user-images.githubusercontent.com/59603451/180108153-57344b7f-ad66-4e68-bdc1-d40dbd03ee7a.png)

Reply:
there is no properties next to $ref in the example you shared...

Reply:
fix it by using "allOf", seems the only way which can address it